2020-02-Spring/0_arrays_matrix/maximum-subarray/maxSubArray.java

```diff
@@ -0,0 +1,18 @@
+public int maxSubArray(int[] nums) {
```

Comment:
This will not compile because a class declaration is missing. Wrap the method in a class declaration, and ensure the class name matches the file name.

I know you've been living in c++ land for the past year or so, but if you're going to write Java, make sure it compiles :+1:

Reply:
Fixed in next commit